packages/extension/src/components/DaDndMessage.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="dnd-message-bar">
+    <p class="do-not-disturb">Do not disturb.</p>
+    <a href="#" class="turn-off" role="button" @click="$emit('dndOff')">Turn Off</a>
+  </div>
+</template>
+
+<style scoped>
```

Comment:
No need for `scoped` styling with such simple component. You can just utilize postcss nesting for this (which is already installed and configured)